webpack.config.renderer.js

```diff
@@ -24,7 +24,7 @@ module.exports = merge(base, {
   },
   module: {
     rules: [{
-      test: /\.jsx$/,
+      test: /\.(js|jsx)?$/,
```

Comment:
I think this change is no longer needed.

Reply:
Doesn't hurt to have it though. As it should not affect anything as such and next time someone would not need to go through this if they need transformations on JS.